homeassistant/components/picnic/const.py

```diff
@@ -0,0 +1,102 @@
+"""Constants for the Picnic integration."""
+from homeassistant.const import CURRENCY_EURO, DEVICE_CLASS_TIMESTAMP
+
+DOMAIN = "picnic"
+
+CONF_API = "api"
+CONF_COORDINATOR = "coordinator"
+CONF_COUNTRY_CODE = "country_code"
+
+COUNTRY_CODES = ["NL", "DE", "BE"]
+ATTRIBUTION = "Data provided by Picnic"
+ADDRESS = "address"
+
+SENSOR_COMPLETED_DELIVERIES = "completed_deliveries"
+SENSOR_TOTAL_ORDERS = "total_orders"
+SENSOR_CART_ITEMS_COUNT = "cart_items_count"
+SENSOR_CART_TOTAL_PRICE = "cart_total_price"
+SENSOR_SELECTED_SLOT_START = "selected_slot_start"
+SENSOR_SELECTED_SLOT_END = "selected_slot_end"
+SENSOR_SELECTED_SLOT_MAX_ODER_TIME = "selected_slot_max_order_time"
+SENSOR_SELECTED_SLOT_MIN_ORDER_VALUE = "selected_slot_min_order_value"
+SENSOR_LAST_ORDER_SLOT_START = "last_order_slot_start"
+SENSOR_LAST_ORDER_SLOT_END = "last_order_slot_end"
+SENSOR_LAST_ORDER_STATUS = "last_order_status"
+SENSOR_LAST_ORDER_ETA_START = "last_order_eta_start"
+SENSOR_LAST_ORDER_ETA_END = "last_order_eta_end"
+SENSOR_LAST_ORDER_DELIVERY_TIME = "last_order_delivery_time"
+SENSOR_LAST_ORDER_TOTAL_PRICE = "last_order_total_price"
+
+SENSOR_TYPES = {
+    SENSOR_COMPLETED_DELIVERIES: {
+        "class": None,
+        "unit": None,
+        "icon": "mdi:truck-delivery",
+    },
+    SENSOR_TOTAL_ORDERS: {
+        "class": None,
+        "unit": None,
+        "icon": "mdi:truck-delivery",
+    },
+    SENSOR_CART_ITEMS_COUNT: {
+        "class": None,
+        "unit": None,
+        "icon": "mdi:format-list-numbered",
+    },
+    SENSOR_CART_TOTAL_PRICE: {
```

Comment:
Home Assistant has the option to have specific entities be disabled by default. You should include that support here so that things that are not critical to a great integration can be opt-in instead of opt-out. Use the `entity_registry_enabled_default` property for this.